### PR TITLE
Move mana_multiplier to enchantments

### PR DESCRIPTION
**Pull request by rty275**
CDDA deprecated mana_multiplier, so it needs to be ported to enchantments. Also I put all the enchantments with their traits instead of in a separate file because I didn't want to make separate enchantments for all the cracked cores

### CODE REVIEW

**Comment by rty275** [comment]
Nevermind, I guess I gotta put the enchantments in their separate file. Working on that now.

**Comment by rty275** [comment]
Okay, dunno why it looks so weird, but that works now with no errors

File: Mana-Cores/traits.json [hunk 1]
```diff
@@ -201,8 +194,6 @@
 		"valid": false,
 		"purifiable": false,
 		"player_display": true,
-		"mana_multiplier": 8,
-		"flags": [ "SILENT_SPELL", "SUBTLE_SPELL" ],
```
Comment (b3brodie) [inline Mana-Cores/traits.json:205]: I think the silent and subtle spell was accidently removed.

Comment (rty275) [reply Mana-Cores/traits.json:205]: So it was, fixed.

**Review by Darius1320 — APPROVED** [review]

**Pull request merged**